app/src/test/java/fi/helsinki/opintoni/web/rest/privateapi/EnrollmentResourceGetStudentCoursesTest.java

```diff
@@ -37,7 +37,7 @@
     private static final String WIKI_COURSE_MATERIAL_URL = "http://wiki.helsinki.fi";
 
     @Test
-    public void thatStudentCoursesAreReturned() throws Exception {
+    public void thatStudentCoursesAreReturnedDefault() throws Exception {
```

Comment:
`Default`?

Reply:
Aivopieru. Tuo overloadasi luokassa käytetyn private yleismetodin.
muutin tuon nimen vastaamaan sen käyttämiä yllä mainitun metodin kutsuparametreja :
` thatStudentCoursesAreReturnedWithCoursePageMaterial `
ja samalla:
`thatCancelledCourseStatusIsReturned` => `thatCancelledCourseStatusIsReturnedWithCoursePageMaterial`

Näin ne vastaavat muita luokassa olevia tuota käyttäviä metodeja nimeämiseltään, tyyliin (`thatStudentCoursesAreReturnedWithWikiMaterial`)